### ISSUE TITLE
Umlauts in title of imported RSS feeds broken

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#7545]
### Expected behavior
When the title element of an RSS item is imported it should show umlauts (ü,ä,ö,ß) as they are.
 
### Actual behavior
The umlauts are encoded and displayed as HTML encoded charcters like `&uuml;`.

### Steps to reproduce the problem

![image](https://user-images.githubusercontent.com/3656054/63362764-a85f0b80-c372-11e9-84a8-29bcd2215be8.png)

Source in the feed:
`<title type="html">Remote Code Execution: Doppelte Hintertür in Webmin</title>`

The issue only exists in the headline (which is a link to the article) - not in the body of the item.

### Friendica version you encountered the problem
2019.09-rc

### Friendica source (git, zip)
git

### PHP version
7.2

### SQL version
10.1 MariaDB

**Comment by AlfredSK** [comment]
Feed source: https://rss.golem.de/rss.php?feed=ATOM1.0&ms=security

Complete feed item:
```
<entry>
       <title type="html">Remote Code Execution: Doppelte Hintertür in Webmin</title>
       <link rel="alternate" type="text/html" href="https://www.golem.de/news/remote-code-execution-doppelte-hintertuer-in-webmin-1908-143311-rss.html"/>
       <published>2019-08-20T11:09:01+02:00</published>
       <updated>2019-08-20T11:09:01+02:00</updated>
       <id>https://www.golem.de/news/remote-code-execution-doppelte-hintertuer-in-webmin-1908-143311-rss.html</id>
       <author>
           <name>Hanno Böck</name>
       </author>
       <summary type="html">In der Systemkonfigurationssoftware Webmin waren offenbar für über ein Jahr Hintertüren, mit denen sich übers Netz Code ausführen lässt. Den Angreifern gelang es dabei offenbar, die Release-Dateien des Projekts zu manipulieren. (&lt;a href=&quot;https://www.golem.de/specials/backdoor/&quot;&gt;Backdoor&lt;/a&gt;, &lt;a href=&quot;https://www.golem.de/specials/serverapps/&quot;&gt;Server-Applikationen&lt;/a&gt;) &lt;img src=&quot;https://cpx.golem.de/cpx.php?class=17&amp;amp;aid=143311&amp;amp;page=1&amp;amp;ts=1566292140&quot; alt=&quot;&quot; width=&quot;1&quot; height=&quot;1&quot; /&gt;</summary>
   </entry>
```

**Comment by MrPetovan** [comment]
Interesting, the HTML entity isn't present in the original title like we had the issue in #7350, but I still thinks it is related to the change in #7534

**Comment by AlfredSK** [comment]
Yep. The items from 3 days ago and older are ok.

**Comment by annando** [comment]
I just wanted to report the same: https://squeet.me/display/962c3e10568a461a58713a303c1ef707452cce29

**Comment by AlfredSK** [comment]
Umm. I just followed your link, @annando
It looks ok. 🤷‍♂️

**Comment by MrPetovan** [comment]
The post title was updated.

**Comment by annando** [comment]
Yeah, he corrected it.

**Comment by Danie10** [comment]
@AlfredSK yes I fixed it manually. Seems to occur from use of apostrophe or quotes in title only - the body is fine as someone else had mentioned.  Other places my RSS feed pulls into do not break the title though.

**Comment by nupplaphil** [comment]
Fixed by #7550
